virttest/rss_client.py

```diff
@@ -170,7 +170,7 @@ def _receive(self, size, timeout=60):
         except socket.error as e:
             raise FileTransferSocketError("Error receiving data from server",
                                           e)
-        return "".join(strs)
+        return b"".join(strs)
```

Comment:
This change would impact `_receive_packet()`, so need to its caller correspondingly.